app/models/work_version.rb

```diff
@@ -117,6 +117,18 @@ def label(id)
             acceptance: true,
             if: :published?
 
+  validates :psu_community_agreement,
+            acceptance: true,
```

Comment:
It took me a while to realize `acceptance: true` rejects falsy things _except_ nil.  That's why we don't need to worry about these with API ingests (like RMD stuff).